go/libraries/doltcore/sqle/enginetest/dolt_engine_test.go

```diff
@@ -1597,6 +1597,28 @@ func TestDoltMergePrepared(t *testing.T) {
 	}
 }
 
+func TestDoltRevert(t *testing.T) {
+	for _, script := range RevertScripts {
+		// harness can't reset effectively. Use a new harness for each script
```

Comment:
Is this caused by how `revert` is implemented? How is this particular situation different from other similar tests?

Reply:
It's not. I'm follow patterns of other tests which make a habit of changing the contents of the branches.